lib/presentation/extensions/room_extension.dart

```diff
@@ -0,0 +1,230 @@
+import 'package:matrix/matrix.dart';
+import 'package:matrix/src/utils/file_send_request_credentials.dart';
+
+extension SendImage on Room {
+
+  static const maxSendingImage = 20;
+
+  Future<String?> sendImageFileEvent(
+    MatrixFile file, {
```

Comment:
how about push the fake event before creating file?

Reply:
But, the placeholder is matrix file, and also the creating file is very fast as i test, so i think no problem at all, maybe when sending multiple images, create files may be slow